src/os/bluestore/Allocator.cc

```diff
@@ -110,6 +115,10 @@ Allocator *Allocator::create(CephContext* cct, string type,
     alloc = new BitmapAllocator(cct, size, block_size, name);
   } else if (type == "avl") {
     return new AvlAllocator(cct, size, block_size, name);
+  } else if (type == "hybrid") {
+    return new HybridAllocator(cct, size, block_size,
+      cct->_conf.get_val<uint64_t>("bluestore_hybrid_alloc_mem_cap"),
+      name);
```

Comment:
this breaks the build of crimson-osd. could you either add `HybridAllocator.cc` to `src/crimson/os/alienstore/CMakeLists.txt` or disable this part if `WITH_ALIEN` is defined?

Reply:
@tchaikov - added to build.